extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -724,16 +436,11 @@ private function renderAvatarLightbox( $userId ) {
 		$user = User::newFromId( $userId );
 
 		$this->setVal( 'defaultAvatars', $this->getDefaultAvatars() );
-		$this->setVal( 'isUploadsPossible',
-			$this->wg->EnableUploads &&
-			$this->wg->User->isAllowed( 'upload' ) &&
-			empty( $this->wg->AvatarsMaintenance )
-		);
 
 		$this->setVal( 'avatarName', $user->getGlobalAttribute( AVATAR_USER_OPTION_NAME ) );
 		$this->setVal( 'userId', $userId );
```

Comment:
Same with userId and avatarName, looks like they're not being used anymore?

Reply:
Thanks! Removed in 5fca599